man/dracut.cmdline.7.asc

```diff
@@ -155,6 +155,10 @@ Misc
     started. If you have hardware, which takes a very long time to announce its
     drives, you might want to extend this value.
 
+**rd.root.overlay**::
```

Comment:
It would be useful to provide guidance how this option is different from systemd.volatile=overlay and what are the tradeoffs ? One difference I noticed is that systemd.volatile=overlay would not leave the mounted filesystem visible. Is that the only difference ?

Reply:
This module runs without systemd. It is used by the FAI project for several years. It seems that systemd.volatile only makes /etc and /var writeable, rd.root.overlay makes the whole / writeable.

Reply:
Thanks for the clarification. I have not realized that systemd is an optional dependency for dracut.